subgraph/networks.json

```diff
@@ -34,5 +34,11 @@
             "address": "0x22410e2a46261a1B1e3899a072f303022801C764",
             "startBlock": 10147523
         }
+    },
+    "mainnet": {
+        "OrderBook": {
+            "address": "0x0eA6d458488d1cf51695e1D6e4744e6FB715d37C",
+            "startBlock": 1041924
```

Comment:
this is not the creation block for this contract https://etherscan.io/tx/0x6d9b4c6cb211f7ba5659783bffa685a7c98cf389f4d1a5f65b3a4adb475c31a5